deployment/CP05MOAS-A6263_Deploy.csv

```diff
@@ -1,57 +1,78 @@
 #AR-28,,AR-28,,CP05MOAS-A6263-06-PARADN000,5,1,2018-03-31T13:18:00,2018-03-31T13:55:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-PARADN-50267,39.91736,-70.76369,,,,mission aborted at 13:36 UTC due to ground fault (GFI); no data
-P10a,,P10a,,CP05MOAS-A6263-00-ENG000000,6,1,2018-04-24T13:05:00,2018-04-24T22:42:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGCON-AUVPNR-06263,40.0167,-71.0967,,,,along shelf surveys
-P10a,,P10a,,CP05MOAS-A6263-01-FLORTN000,6,1,2018-04-24T13:05:00,2018-04-24T22:42:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-FLORTN-01645,40.0167,-71.0967,,,,along shelf surveys
-P10a,,P10a,,CP05MOAS-A6263-02-DOSTAN000,6,1,2018-04-24T13:05:00,2018-04-24T22:42:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-DOSTAN-02901,40.0167,-71.0967,,,,along shelf surveys
-P10a,,P10a,,CP05MOAS-A6263-03-CTDAVN000,6,1,2018-04-24T13:05:00,2018-04-24T22:42:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-CTDAVN-00424,40.0167,-71.0967,,,,along shelf surveys
-P10a,,P10a,,CP05MOAS-A6263-04-NUTNRN000,6,1,2018-04-24T13:05:00,2018-04-24T22:42:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-NUTNRN-01109,40.0167,-71.0967,,,,along shelf surveys
-P10a,,P10a,,CP05MOAS-A6263-05-ADCPAN000,6,1,2018-04-24T13:05:00,2018-04-24T22:42:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-ADCPAN-22076,40.0167,-71.0967,,,,along shelf surveys
-P10a,,P10a,,CP05MOAS-A6263-06-PARADN000,6,1,2018-04-24T13:05:00,2018-04-24T22:42:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-PARADN-50267,40.0167,-71.0967,,,,along shelf surveys
+WJ20180423,,WJ20180423,,CP05MOAS-A6263-00-ENG000000,6,1,2018-04-24T13:05:00,2018-04-24T22:42:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGCON-AUVPNR-06263,40.0167,-71.0967,,,,along-shelf survey
+WJ20180423,,WJ20180423,,CP05MOAS-A6263-01-FLORTN000,6,1,2018-04-24T13:05:00,2018-04-24T22:42:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-FLORTN-01645,40.0167,-71.0967,,,,along-shelf survey
+WJ20180423,,WJ20180423,,CP05MOAS-A6263-02-DOSTAN000,6,1,2018-04-24T13:05:00,2018-04-24T22:42:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-DOSTAN-02901,40.0167,-71.0967,,,,along-shelf survey
+WJ20180423,,WJ20180423,,CP05MOAS-A6263-03-CTDAVN000,6,1,2018-04-24T13:05:00,2018-04-24T22:42:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-CTDAVN-00424,40.0167,-71.0967,,,,along-shelf survey
+WJ20180423,,WJ20180423,,CP05MOAS-A6263-04-NUTNRN000,6,1,2018-04-24T13:05:00,2018-04-24T22:42:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-NUTNRN-01109,40.0167,-71.0967,,,,along-shelf survey
+WJ20180423,,WJ20180423,,CP05MOAS-A6263-05-ADCPAN000,6,1,2018-04-24T13:05:00,2018-04-24T22:42:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-ADCPAN-22076,40.0167,-71.0967,,,,along-shelf survey
+WJ20180423,,WJ20180423,,CP05MOAS-A6263-06-PARADN000,6,1,2018-04-24T13:05:00,2018-04-24T22:42:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-PARADN-50267,40.0167,-71.0967,,,,along-shelf survey

```

Comment:
@cdobs These cruise numbers (WJ20180423) do not match the ones in your AUV Tracking 2.0 spreadsheet. Based on the date in this cruise number, I think it is correct here and incorrect on your spreadsheet (which says "P10a (WJ20180705)"). Can you verify?

Reply:
Yup, it should be WJ20180423, I fixed the spreadsheet!